less/pages/thank-you.less

```diff
@@ -12,3 +9,77 @@
     display: none;
   }
 }
+.thank-you-page,
+.share-page {
```

Comment:
I moved these into specific pages, instead of having it live in shared.

new-flow did this but regular did not. So moving it over, I kept that.